### DISCUSSION TITLE
join pool

### DISCUSSION THREAD

**Discussion by nabiesl** [Chia-Network/chia-blockchain#7405] [Support]
Hello, good morning, despite the pool in the new version, we have to remove the plots made from the cubes and create new plots?
![join-chia-pool](https://user-images.githubusercontent.com/86523027/125153767-e3a38880-e10a-11eb-865d-89127ecfcca8.png)

**Comment by rhernaus** [comment]
You can still solo farm your 'legacy' plots. 
If you want to use the official Chia pooling protocol that is shown in your screenshot then you have to make new plots with your Plot NFT encoded in them.

**Comment by nabiesl** [comment]
What is the reason for these names in the pool section? Amaranth aardwolf & scarlet squirrel

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
